tests/test_hodgkin.py

```diff
@@ -276,34 +276,40 @@ def test_has_ontology_annotation(self, model):
     def test_get_equations_for(self, graph, model):
         # Get equations for membrane_fast_sodium_current both ordered and unordered
         membrane_fast_sodium_current = model.get_symbol_by_ontology_term(OXMETA, "membrane_fast_sodium_current")
-        equations = model.get_equations_for([membrane_fast_sodium_current])
-        unordered_equations = model.get_equations_for([membrane_fast_sodium_current], False)
-        # There should be 4 in this model
-        assert len(equations) == len(unordered_equations) == 4
+        membrane_stimulus_current_offset = model.get_symbol_by_ontology_term(OXMETA, "membrane_stimulus_current_offset")
+        membrane_stimulus_current_period = model.get_symbol_by_ontology_term(OXMETA, "membrane_stimulus_current_period")
+
+        equations = model.get_equations_for([membrane_fast_sodium_current, membrane_stimulus_current_offset])
+        input_ordered_equations = model.get_equations_for(
+            [membrane_fast_sodium_current, membrane_stimulus_current_offset,
+             membrane_stimulus_current_offset, membrane_stimulus_current_period],
+            True, [membrane_stimulus_current_period])
+
+        # There should be 10 in this model
+        assert len(equations) == len(input_ordered_equations) == 5
         # Each equation should be both in the ordered and unordered equations
         for eq in equations:
-            assert eq in unordered_equations
-        for eq in unordered_equations:
+            assert eq in input_ordered_equations
+        for eq in input_ordered_equations:
             assert eq in equations
-
         # Expected equations
         ref_eq = [sympy.Eq(sympy.Dummy('membrane$E_R'), sympy.numbers.Float(-75.0)),
+                  sympy.Eq(sympy.Dummy('membrane$stim_start'), sympy.numbers.Float(10.0)),
                   sympy.Eq(sympy.Dummy('sodium_channel$E_Na'),
                            sympy.add.Add(sympy.Dummy('membrane$E_R'), sympy.numbers.Float(115.0))),
                   sympy.Eq(sympy.Dummy('sodium_channel$g_Na'), sympy.numbers.Float(120.0)),
                   sympy.Eq(sympy.Dummy('sodium_channel$i_Na'),
                            sympy.Dummy('sodium_channel_m_gate$m') ** 3.0 * sympy.Dummy('sodium_channel$g_Na') *
                            sympy.Dummy('sodium_channel_h_gate$h') * (sympy.Dummy('membrane$V') -
-                           sympy.Dummy('sodium_channel$E_Na')))
-                  ]
```

Comment:
I would leave the close bracket on a separate line as it makes it much clearer when doing a diff what has actually changed

Reply:
ok